pkgs/top-level/python-packages.nix

```diff
@@ -2796,6 +2796,13 @@ in {
 
   hydra-check = callPackage ../development/python-modules/hydra-check { };
 
+  hydrus = pkgs.libsForQt5.callPackage ../development/python-modules/hydrus {
```

Comment:
I'm guessing hydrus is really not intended to be used as a library, so maybe better to not add it to python-packages.nix or to development/python-modules at all - maybe under pkgs/applications/graphics if I understand the application right?

Reply:
I was wondering the same thing.
Wasn't sure if it would be better under there or `servers`, etc.

Reply:
Settled on the initially suggested `pkgs/applications/graphics`